portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Workspace.testcase

```diff
@@ -321,6 +321,9 @@ definition {
 			ACUtils.closeModal();
 		}
 
+		task ("Click on different setting tabs") {
+			ACSettings.goToUsage();
+
```

Comment:
Hey @calvinctvo I think you are missing a closing bracket here.

Reply:
thank you @manuferreira